isis/src/control/apps/cnetwinnow/cnetwinnow.h

```diff
@@ -0,0 +1,21 @@
+#ifndef spiceinit_h
+#define spiceinit_h
+
+#include <vector>
```

Comment:
Are these imports necessary? Specifically vector, QString, QList, QVector

Reply:
No, they are not. I will remove them